### PR TITLE
feat(android): add apk & aab subcommands to build apk/aab

### PR DESCRIPTION
**Pull request by amrbashir**
#### TODO:
- [x] universal apk/aab should build include only specified targets and fallback to all

### CODE REVIEW

File: src/android/aab.rs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                app.name(),
+                build_ty,
+                profile,
+            )))
+            .to_path_buf(),
+        );
+    }
+
+    println!();
+    println!("Finished bundling AAB(s):");
```
Comment (wusyong) [inline src/android/aab.rs]:
```suggestion
    println!("\nFinished bundling AAB(s):");
```

File: src/android/apk.rs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                app.name(),
+                profile,
+                profile,
+            )))
+            .to_path_buf(),
+        );
+    }
+
+    println!();
+    println!("Finished building APK(s):");
```
Comment (wusyong) [inline src/android/apk.rs]:
```suggestion
    println!("\nFinished building APK(s):");
```

File: README.md [hunk 3]
```diff
@@ -45,16 +39,14 @@ To start a new project, all you need to do is make a directory with a cute name,
...
-| name      | info                                                                                                                              |
-| --------- | --------------------------------------------------------------------------------------------------------------------------------- |
-| bevy      | Minimal Bevy project derived from [sprite](https://github.com/bevyengine/bevy/blob/master/examples/2d/sprite.rs) example          |
-| bevy-demo | Bevy [breakout](https://github.com/bevyengine/bevy/blob/master/examples/game/breakout.rs) demo                                    |
-| wgpu      | Minimal wgpu project derived from [hello-triangle](https://github.com/gfx-rs/wgpu-rs/tree/master/examples/hello-triangle) example |
-| winit     | Minimal winit project derived from [window](https://github.com/rust-windowing/winit/blob/master/examples/window.rs) example          |
+| name      | info                                  |
+| --------- | ------------------------------------- |
+| wry       | Minimal wry project |
+| tauri     | Minimal tauri project |
```
Comment (wusyong) [inline README.md:47]: Could we take this chance to remove this template?
Or you think it's still better to have one?

Comment (amrbashir) [reply README.md:47]: I think there is still some people using that template, so lets keep it for now.

**Review by wusyong — DISMISSED** [review]

**Comment by lucasfernog** [comment]
I can't test right now but this looks good!

**Comment by lucasfernog** [comment]
Btw the `run` command also uses some apk/aab functionality in the Device struct.

**Comment by amrbashir** [comment]
ah let me extract them to some common module or something

**Comment by amrbashir** [comment]
should be fine now, device module outsources apk/aab building to apk/aab module respectively.

**Pull request merged**